File: problemas/matematica/1761/1761.go
package main

import (
	"bufio"
	"fmt"
	"math"
	"os"
)

func main() {
	var a, b, c float64

	scanner := bufio.NewScanner(os.Stdin)

	pi := 3.141592654
	for scanner.Scan() {
		fmt.Sscanf(scanner.Text(), "%f %f %f\n", &a, &b, &c)
		fmt.Printf("%.2f\n", 5.0 * (math.Tan(a * pi / 180.0) * b + c))
    }
}

File: problemas/matematica/1761/1761.kt
import java.lang.Math
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    val pi = 3.141592654
    while (sc.hasNext()) {
        val a = sc.nextDouble()
        val b = sc.nextDouble()
        val c = sc.nextDouble()

        println(String.format("%.2f", 5.0 * (Math.tan(a * pi / 180.0) * b + c)))
    }
}
